HW/spark.py

```diff
@@ -0,0 +1,104 @@
+import sys
+
+from pyspark.sql import Row,SparkSession
+from pyspark.sql.types import (
+    BooleanType,
+    DoubleType,
+    IntegerType,
+    StringType,
+    StructField,
+    StructType,
+)
+
+def main():
+    appName = "Baseball"
+    master = "local"
+    # Setup Spark
+    spark = SparkSession.builder.master("local[*]").getOrCreate()
```

Comment:
I think this is where we need to specify the jdbc driver path but I might be wrong